data/baibhav-tiwari.json

```diff
@@ -3,13 +3,18 @@
   "displayStatsPublic": true,
   "type": "personal",
   "bio": "**Web3** | **Web-developer** | **React** | **Open_Source**",
-
   "tags": ["Open Source", "Javascript", "ReactJS", "Web3"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/BaibhavTiwari11" },
+    { "icon": "FaGithub", "url": "https://github.com/baibhav-tiwari" },
+    { "icon": "SiHashnode", "url": "https://www.linkedin.com/in/baibhavtiwari" }
+   ],
   "links": [
     {
       "name": "Twitter",
       "url": "https://twitter.com/BaibhavTiwari11",
       "icon": "FaTwitter"
+baibhavr"
```

Comment:
this line needs to fix.

Reply:
just created a new one with all the existing changes and in a correct manner.
thank you for helping me out